worker_manage.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import print_function
+
+import click
+from flask.cli import FlaskGroup
+
+import dataset_eval.evaluate
+import hl_extractor.hl_calc
+import webserver
+
+cli = FlaskGroup(add_default_commands=False, create_app=webserver.create_app)
+
+
+@cli.command('hl_extractor', help='High-level extractor tool')
+@click.option('-t', default=1, type=int)
```

Comment:
Docstrings for this and the other command would be good to have.

Reply:
Also, maybe using `threads` instead of `t` would be better.

Reply:
I moved the help arguments to a docstring (click uses these as help if not explicitly set) and also cleaned up the docs in the main manage.py.

Added --threads as a long argument format.

Removed all create_app that wasn't needed